pkgs/by-name/pl/plant-it/package.nix

```
@@ -0,0 +1,58 @@
+{
+  maven,
+  jdk21_headless,
+  makeWrapper,
+  lib,
+  callPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+let
+  version = "0.10.0";
+  src = fetchFromGitHub {
+    owner = "MDeLuise";
+    repo = "plant-it";
+    tag = version;
+    hash = "sha256-QnujZecUu7bzllSsrLH6hSZMaWeOUXBrSZ5rbT56pDM=";
+  };
+  frontend = callPackage (import ./frontend.nix src version) { };
+  backend-src = "${src}/backend";
+in
+maven.buildMavenPackage rec {
+  pname = "plant-it";
+  inherit version;
+  src = backend-src;
+
+  mvnHash = "sha256-3YQOZMXMI6BrHkqud2OKColJWbDXfwnAwRifYxbleqI=";
+  nativeBuildInputs = [
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```
You probably want `makeBinaryWrapper`, as it's the [preferred implementation](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper) of `makeWrapper`.